test/nn/aggr/test_quantile.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import torch
+
+from torch_geometric.nn import MedianAggregation, QuantileAggregation
+
+x = torch.tensor([
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [9, 0, 1],
+    [2, 3, 4],
+    [5, 6, 7],
+    [8, 9, 0],
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+]).float()
+
+
+@pytest.mark.parametrize('index,result,dim', [
```

Comment:
[nit] I feel this is pretty verbose, personally would prefer just defining in the function body

Reply:
I did it as you suggested in the first place, but then switched to `parametrize` to know which test case failed, while using a for loop inside the function body would only show a single `F` during `pytest`. Any way, I could change it, if you prefer the other way!

Reply:
Ah yes that is true. I don't really have a strong opinion here, have just seen the other way in most of the codebase. Probably not worth changing.